config/locales/en/admin.yml

```diff
@@ -114,6 +114,7 @@ en:
         phase: Phase
         phases_caption: "Phases"
         duration: "Duration"
+        duration_help_text: "(Informative field)"
```

Comment:
I think we can remove the parenthesis since we don't use them in other hints in the application :thinking:.

"Informative field" sound a bit funny; I was thinking of "For information purposes only", but I'm not sure whether that's too  long :thinking:.

Reply:
I've just realized the whole phases table is used for information purposes only, since it's possible to disable the active phase or select a disabled phase as the active one. So maybe we could add an explanatory text before the table and then we wouldn't have to make the text particularly short :thinking:.

Just an idea; I'm not sure at all whether it's a good one.

I've also noticed that the hint for the "phase enabled" field says "This phase will be public in the budget's phases timeline, as well as active for any other purpose". I haven't found any other purpose :sweat_smile:; is there anything I'm missing? If there isn't any other purpose, we could remove this part of the sentence.